rm-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/client/CompactView.java

```diff
@@ -71,39 +68,27 @@
 
     private Layout root;
 
-    /* displays nodes as a compact grid */
-    private FlowPanel flow;
-
-    boolean _borderSwitch;
+    private boolean _borderSwitch;
 
-    boolean _doNotScroll;
+    private boolean _doNotScroll;
 
     private static Layout globalHover = null;
 
-    /*
-     * unique names of the nodesources/hosts/nodes currently held by the FlowLayout
-     * helps figuring out the position of each element since FlowLayout doesnt allow it
-     */
-    private LinkedList<String> curTiles = null;
+    /* displays nodes as a compact grid */
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this field "_borderSwitch" to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00116)